src/ui/mod.rs

```diff
@@ -470,9 +470,10 @@ where
             }
           }
           album_artist.push_str(&format!(
-            "{} - {}",
+            "{} - {} ({})",
             item.name.to_owned(),
-            create_artist_string(&item.artists)
+            create_artist_string(&item.artists),
+            item.album_type.as_deref().unwrap()
```

Comment:
I don't think it's likely that an artist album has no type (I certainly haven't been able to find any), but I still don't love the naked `unwrap`. Can we change instead to something like `unwrap_or("unknown")`

Reply:
I'm happy with that. Pushed change in commit: 75f107f. Marking as Resolved.